Tests/SwiftSyntaxTest/AbsolutePosition.swift

```diff
@@ -101,13 +101,14 @@ public class AbsolutePositionTestCase: XCTestCase {
   public func testTrivias() {
     let idx = 5
     let root = createSourceFile(idx + 1)
-    XCTAssertEqual(root.leadingTrivia!.count, 3)
-    XCTAssertEqual(root.trailingTrivia!.count, 0)
+    XCTAssertEqual(3, root.leadingTrivia!.count)
+    XCTAssertEqual(0, root.trailingTrivia!.count)
     let state = root.statements[idx]
-    XCTAssertEqual(state.leadingTrivia!.count, 3)
-    XCTAssertEqual(state.trailingTrivia!.count, 1)
-    XCTAssertEqual(state.leadingTrivia!.byteSize + state.trailingTrivia!.byteSize
-      + state.byteSizeAfterTrimmingTrivia, state.byteSize)
+    XCTAssertEqual(3, state.leadingTrivia!.count)
+    XCTAssertEqual(1, state.trailingTrivia!.count)
```

Comment:
Again, expected value as the second parameter.

Reply:
Done.